process.go

```diff
@@ -231,8 +231,16 @@ func runTestSuite(ts *TestSuite, detailsLevel string) {
 }
 
 func runTestCase(ts *TestSuite, tc *TestCase, l *log.Entry, detailsLevel string) {
+	l.Infof("Init context")
```

Comment:
I put info like logs below :/
l.Infof("start")
l.Infof("end")

Debug for all ?